libs/books/feature/src/lib/undo-action-snackbar-serivice/undo-action.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable, OnDestroy } from '@angular/core';
```

Comment:
Do not write service file since we can handle these in effects. Please remove this file move logic to effects